### PR TITLE
Created student profile box component

### PR DESCRIPTION
**Pull request by crystalleesf**
Separate PR for the Student Profile Box component

image

### CODE REVIEW

File: src/components/StudentProfileBox/StudentProfileBox.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import '../../common/vars.css';
+import { Button, Col } from 'react-bootstrap';
+import styles from './StudentProfileBox.module.css';
+
+const StudentProfileBox = ({ studentName }) => {
+  return (
+    <>
+      <Col className="d-flex justify-content-center align-items-center">{studentName}</Col>
+      <Col className="d-flex justify-content-center align-items-center">
+        <Button className={styles['view-profile-btn']}>View Profile</Button>
```
Comment (anikait-rao) [inline src/components/StudentProfileBox/StudentProfileBox.jsx:12]: I think you should also have a backend request here for the student info - you will need the `studentId` in order for the "view profile" button to work properly

**Review by allenxluo — COMMENTED** [review]
image

Looks like this when I run it. Are you guys seeing it differently (as seperate columns)

File: src/components/StudentProfileBox/StudentProfileBox.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../common/vars.css';
+import { Button, Col } from 'react-bootstrap';
+import styles from './StudentProfileBox.module.css';
+
+const StudentProfileBox = ({ studentName }) => {
+  return (
+    <>
```
Comment (anikait-rao) [inline src/components/StudentProfileBox/StudentProfileBox.jsx]: Please include the border of the box

**Review by anikait-rao — CHANGES_REQUESTED** [review]
See comments

File: src/components/StudentProfileBox/StudentProfileBox.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (anikait-rao) [inline src/components/StudentProfileBox/StudentProfileBox.jsx:1]: Also, the component is appearing vertically for me. Not sure if this is an issue on my end? 
image

**Comment by crystalleesf** [comment]
@anikait-rao @allenxluo yeah i just realized that we did the Row part in another component. I modified it so the Row is now part of this component. lmk if it works now
Screen Shot 2022-04-06 at 10 17 20 AM

**Review by anikait-rao — APPROVED** [review]
nice job

**Pull request merged**